spec.bs

```diff
@@ -369,19 +749,74 @@ A <dfn export>fenced frame config</dfn> is a struct with the following [=struct/
 <div algorithm>
   The <dfn method for=Fence>reportEvent(<var ignore>event</var>)</dfn> method steps are:
 
-  1. <span class=XXX>Fill this out!</span>
+  1. Let |instance| be [=this=]'s [=relevant global object=]'s [=associated Document=]'s [=node
+     navigable=]'s [=navigable/traversable navigable=]'s [=fenced frame config instance=].
+
+     <span class=XXX>This and the below references should point to an actual member on [=traversable
+     navigable=], not just the type.</span>
```

Comment:
I put this as a TODO here that we should follow-up on quickly, but feel free to address this before we land if you want. Basically I think inside [this section](https://wicg.github.io/fenced-frame/#traversable-navigables) we should introduce a new member called "fenced frame config instance" or whatever is similar to what we have on `DocumentLoader` where this can live. That way here and elsewhere we have a concrete thing to initialize, assign, and address wherever we need.

Reply:
Agree. Where should it live? If we didn't have to deal with urn iframes I think it could just be attached to the fenced content navigable or whatever, but we probably have to attach it to the root.

Reply:
That reminds me: this part about fenced frame config mappings is probably wrong for urn iframes (though it is wrong in the same way that our implementation is "wrong"):

```
Each [=traversable navigable=] has a <dfn for="traversable navigable" export>fenced frame config
mapping</dfn>, which is a new [=fenced frame config mapping=].
```

Reply:
So the fenced navigable is indeed a traversable navigable, per https://wicg.github.io/fenced-frame/#fenced-navigable-container-fenced-navigable. That's the root, essentially `content::Page` / `content::FencedFrame` so the member should probably live there, and iframes can climb the tree to reference it. That's basically what the implementation does, right?

Reply:
Ah yeah... hmm OK I think for now we should make it a normal member of "navigable" (basically FTN) with some strong Note saying that once URN iframes are gone this will exclusively live on "traversable navigable".